From: Sean Finney <seanius@debian.org>
Date: Tue, 29 Nov 2011 18:24:26 +0100
Subject: send_nsca.cfg

default to /etc/send_nsca.cfg instead of ./send_nsca.cfg
---
 src/send_nsca.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/send_nsca.c b/src/send_nsca.c
index a75da0f..b329cb4 100644
--- a/src/send_nsca.c
+++ b/src/send_nsca.c
@@ -25,7 +25,7 @@ time_t start_time,end_time;
 int server_port=DEFAULT_SERVER_PORT;
 char server_name[MAX_HOST_ADDRESS_LENGTH];
 char password[MAX_INPUT_BUFFER]="";
-char config_file[MAX_INPUT_BUFFER]="send_nsca.cfg";
+char config_file[MAX_INPUT_BUFFER]="/etc/send_nsca.cfg";
 char delimiter[2]="\t";
 char block_delimiter[2]=BLOCK_DELIMITER;
 
